registry/remote/example_test.go

```diff
@@ -451,6 +479,51 @@ func ExampleRepository_Fetch_artifactReferenceManifest() {
 	// {"mediaType":"application/vnd.cncf.oras.artifact.manifest.v1+json","artifactType":"example/signature","blobs":null,"subject":{"mediaType":"application/vnd.oci.image.manifest.v1+json","digest":"sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b","size":24}}
 }
 
+// ExampleRepository_Fetch_blobsOfArtifactManifest gives an example of pulling the blobs
+// of an artifact manifest.
+func ExampleRepository_Fetch_blobsOfArtifactManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+
+	// Fetch the artifact manifest by digest. The digest can be obtained from Referrers API.
```

Comment:
nit: Sorry, I read through this again and realized that we don't need to call out where the digest is from. As this can be a standalone ORAS artifact.